appendices/VK_KHR_external_memory.txt

```diff
@@ -137,8 +137,8 @@ Options for defining this transition include:
 A new structure has the advantage that the type of external transition can
 be described in as much detail as necessary.
 However, there is not currently a known need for anything beyond
-differentiating external vs.
-internal accesses, so this is likely an over-engineered solution.
+differentiating external versus internal accesses, so this is likely an
```

Comment:
Better dealt with by adding vs. to endAbbrev in scripts/reflow.py.

Reply:
Maybe.
Probably could also be reworded to english instead of using cutesy latin words.

Reply:
"versus" and its abbreviation are inherited from Middle English. We'll continue to use it.

Reply:
@oddhack Reworded one case, added exception to the script too.

Yes, latin is part of many modern languages. What I mean is that in 90 % of the cases it still looks like "look I know latin" rather than being the best way to phrase something.